app/coffee/HttpController.coffee

```diff
@@ -103,20 +103,28 @@ module.exports = HttpController =
 			logger.log project_id: project_id, doc_id: doc_id, "flushed doc via http"
 			res.send 204 # No Content
 
-	flushAndDeleteDoc: (req, res, next = (error) ->) ->
+	deleteDoc: (req, res, next = (error) ->) ->
 		doc_id = req.params.doc_id
 		project_id = req.params.project_id
-		logger.log project_id: project_id, doc_id: doc_id, "deleting doc via http"
+		flush = req.body.flush ? true
```

Comment:
When there's an action which is destructive I generally like to have an explicit positive action for it, rather than a negative one, to make the consequences clearer: e.g.  having to set  `hardDelete:true` might give more pause for thought than setting `flush:false`.  What do you think?

Most of the existing code uses a query string for these actions like `?force=true` - maybe use a query string for consistency?

Although I have used coffeescript for a long time I wasn't familiar with the meaning of `a ? b` and had to look it up.  To keep the code simple, I think the log message could go inside each branch of the `if` statement, with a different message for the new case e.g. "_hard deleting doc from redis - content will be lost_"

Reply:
I agree. I used `flush` because I found it more informative than `hardDelete`. How about `skipFlush`?